sal-service/src/main/java/org/ow2/proactive/sal/service/service/JobService.java

```diff
@@ -532,6 +532,19 @@ public Boolean killJob(String sessionId, String jobId) throws NotConnectedExcept
         return result;
     }
 
+    /**
+     * Get a specific job skeleton
+     * @param sessionId A valid session id
+     * @param taskId A valid task identifier
+     * @return A Job instance
+     */
+    public Task getTask(String sessionId, String taskId) throws NotConnectedException {
+        if (!paGatewayService.isConnectionActive(sessionId)) {
+            throw new NotConnectedException();
+        }
+        return repositoryService.getTask(taskId);
```

Comment:
it seems this one will be silent in the if does not exist in the db, I think it's better to throw and exception

Reply:
Sorry, I marked it as resolved by mistake.
This is done this way to be consistent with other getters endpoints. If the job, task, deployment ... is not found an empty response is sent.

Reply:
yup I guess this is a work for later, to add at least a warning, for now it can be handled from the other end